Cargo.toml

```diff
@@ -256,6 +257,7 @@ tree-sitter-python = "0.20.2"
 tree-sitter-racket = { git = "https://github.com/zed-industries/tree-sitter-racket", rev = "eb010cf2c674c6fd9a6316a84e28ef90190fe51a" }
 tree-sitter-ruby = "0.20.0"
 tree-sitter-rust = "0.20.3"
+tree-sitter-scss = { git = "https://github.com/serenadeai/tree-sitter-scss", rev = "c478c6868648eff49eb04a4df90d703dc45b312a" }
```

Comment:
Does this grammar need to be added? I can't see it being used elsewhere in the PR.

Reply:
I was testing it out to see if solved the highlighting issues within the `style` tag. I'll remove it.